src/metabase/driver/sql_jdbc/sync/describe_database.clj

```diff
@@ -32,6 +32,23 @@
    (fn [^ResultSet rs]
      #(.getString rs "TABLE_SCHEM"))))
 
+(defn syncable-schemas-for-db
```

Comment:
I placed it here so we can further re-use it. I can't reuse it in the default imp of `describe-database` because `active-tables` does not take a `db`.